bibliography.bib

```diff
@@ -309,6 +309,48 @@ @article{knight2017measuring
   publisher={Springer}
 }
 
+@article{knight2013selfish,
+  title={Selfish routing in public services},
+  author={Knight, Vincent A and Harper, Paul R},
+  journal={European Journal of Operational Research},
+  volume={230},
+  number={1},
+  pages={122--132},
+  year={2013},
+  publisher={Elsevier}
+}
+
+@article{clarey2014ambulance,
+  title={Ambulance handovers: can a dedicated ED nurse solve the delay in ambulance turnaround times?},
+  author={Clarey, A and Allen, M and Brace-McDonnell, S and Cooke, MW},
+  journal={Emergency Medicine Journal},
+  volume={31},
+  number={5},
+  pages={419--420},
+  year={2014},
+  publisher={BMJ Publishing Group Ltd and the British Association for Accident~…}
+}
+
+@inproceedings{koutsoupias1999worst,
+  title={Worst-case equilibria},
+  author={Koutsoupias, Elias and Papadimitriou, Christos},
+  booktitle={Annual Symposium on Theoretical Aspects of Computer Science},
+  pages={404--413},
+  year={1999},
+  organization={Springer}
+}
+
+@article{deo2011centralized,
+  title={Centralized vs. decentralized ambulance diversion: A network perspective},
+  author={Deo, Sarang and Gurvich, Itai},
+  journal={Management Science},
+  volume={57},
+  number={7},
+  pages={1300--1319},
+  year={2011},
+  publisher={INFORMS}
+}
+
 ###################
 ## NEWS ARTICLES ##
 ###################
```

Comment:
I believe `s/#/%` is more robust here.

Reply:
What do you mean? Should I remove that comment? Or should I just say:

`# NEWS ARTICLES`

Reply:
No I believe technically you're using `#` for comments which could be brittle for the TeX compiler which would look for `%` as a comment.

Reply:
I think the .bib file doesn't support comments (might be wrong though). You can type anything outside of environment and wouldn't make a difference:

```
@article{key
   ...
}
```

But I guess I can use `%` for consistency